.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '00 19 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-issue-message: |
+            This issue is marked as stale due to requiring feedback for 30 days. Please respond within 5 days or this issue will be closed due to inactivity.
+          close-issue-message: |
+            This issue was closed due to inactivity. If you can still reproduce this bug, please comment with the following information:
+            1. Full output of running `msbuild --version` and `dotnet --version` (if applicable) on a developer command prompt.
```

Comment:
This seems unlikely to be generally helpful.

Reply:
Saw something similar to this in another repo's stale yml file. They had this awesome section that said "if this still fails on latest/master..." which we can't tell people, so I can see why this isn't super helpful.